### PR TITLE
Update SRCREV for atf, u-boot and kernel

### PR DESCRIPTION
**Pull request by rebeccasf**

### CODE REVIEW

File: recipes-kernel/linux/linux-altera-lts_5.4.bb [hunk 1]
```diff
@@ -3,7 +3,7 @@ LINUX_VERSION_SUFFIX = "-lts"
 
 LIC_FILES_CHKSUM = "file://COPYING;md5=bbea815ee2795b2f4230826c0c6b8814"
 
-SRCREV = "7299712b69b1869ddaf1c143c9264b69351d9e2b"
+SRCREV = "be87fdd9e66c1bdea063b7771af74fdb2b389571"
```
Comment (kraj) [inline recipes-kernel/linux/linux-altera-lts_5.4.bb:6]: should it also bump LINUX_VERSION on line 1?

Comment (rebeccasf) [reply recipes-kernel/linux/linux-altera-lts_5.4.bb:6]: I am checking the test result of new version bump, will submit another pull request for version bump

Comment (kraj) [reply recipes-kernel/linux/linux-altera-lts_5.4.bb:6]: thanks I will wait for you you to report back

**Comment by rebeccasf** [comment]
updated the pull request with kernel version bump to 5.4.64

**Review by kraj — APPROVED** [review]

**Pull request merged**